index.html

```diff
@@ -1,199 +1,315 @@
-                        <h3>languages</h3>
-                        <ul class="my-languages">
-                            <li>Javascript</li>
-                            <li>Ruby</li>
-                            <li>HMTL</li>
-                            <li>CSS</li>
-                        </ul>
+    
+            </section>
+        </article>
+        <article class="card-holder">
+            <section>
+                <div class="card">
+                    <div class="empty-box">
+    
+                    </div>
+                    <div class="multi-post-stories">
+                        <h3>Multi-Post Stories Gain + Glory</h3>
+                        <div class="programing-languages">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>Javascript</li>
+                                <li>HTML</li>
+                            </ul>
                         </div>
+                        <button class="green-button">See project</button>
                     </div>
+    
+            </section>
+        </article>
+     </div>
+    <section>
+        <div class="right-design r-sign"><img src="images-2/right.svg" alt="right-design"></div>
+        <div class="me-section">
+            <div class="empty--box">
+                <div class="myself">
+                    <div class="head">
+                        <h2>About me</h2>
+                    </div>
+                </div>
+            </div>
 
-                    <div>
-                        <div class="square-card">
-                            <div class="language-img">
-                            <img src="images-2/second-icon.svg" alt="Framewokrs">
-                            </div>
-                        <h3>frameworks</h3>
-                        <ul class="my-languages">
-                            <li>React.js</li>
-                            <li>Ruby on Rails</li>
-                            <li>RSpec</li>
-                            <li>Capybara</li>
-                            <li>Selenium</li>
-                        </ul>
-                        </div>
+            <div class="paragraph">
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+            </div>
+            <div>
+                <button class="Get-my-resume">Get my resume</button>
+            </div>
+            <div class="l-design">
+                <img src="images-2/left.svg" alt="left design">
+
+            </div>
+            <div class="all-square-cards"><div>
+                <div class="square-card first-sqaure-card">
+                    <div class="language-img">
+                        <img src="images-2/first-icon.svg" alt="languages">
```

Comment:
- [ ] According to your previouse reviewer please  background-position is not look good, please must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) follow the design specification and update accordingly .
- [ ] 
![image](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/45629d22-14a4-4d22-bf15-faf76adfc10c)